### ISSUE TITLE
faulty documentation on easybuild config file value specification

### ISSUE THREAD

**Issue by rmfrankch** [hpcugent/easybuild#258]
documentation 2.9.0

chapter: Configuring EasyBuild
section: Templates and constants supported in configuration files
example:
robot-paths = %(repositorypath)s:%(DEFAULT_ROBOT_PATHS)s

This references repositorypath which is only defined in a section following this definition.

Result: 
ERROR: Failed to parse configuration options: Bad value substitution:
	section: [basic]
	option : robot-paths
	key    : repositorypath
	rawval : :

**Comment by boegel** [comment]
@rmfrankch thanks for the bug report

This does work, so I'll update documentation accordingly:

```
$ cat test.cfg
[basic]
repositorypath = /home/example/easybuild/easyconfigs_archive
robot-paths = %(repositorypath)s:%(DEFAULT_ROBOT_PATHS)s

$ eb --show-config --configfiles test.cfg
#
# Current EasyBuild configuration
# (C: command line argument, D: default value, E: environment variable, F: configuration file)
#
buildpath             (D) = /Users/kehoste/.local/easybuild/build
configfiles           (C) = /Users/kehoste/.config/easybuild/config.cfg, test.cfg
git-working-dirs-path (E) = /Users/kehoste/work
github-user           (E) = boegel
installpath           (D) = /Users/kehoste/.local/easybuild
modules-tool          (E) = Lmod
repositorypath        (F) = /home/example/easybuild/easyconfigs_archive
robot-paths           (F) = /home/example/easybuild/easyconfigs_archive, /Users/kehoste/work/easybuild-easyconfigs/easybuild/easyconfigs
sourcepath            (D) = /Users/kehoste/.local/easybuild/sources
```

**Comment by boegel** [comment]
fixed in #258
